Tests/CacheAdvanceTests/CacheAdvanceTests.swift

```diff
@@ -96,114 +96,31 @@ final class CacheAdvanceTests: XCTestCase {
-        // The 00011010 value is a larger value what we intended to write, which would lead to file corruption.
-        try header.updateOffsetInFileAtEndOfNewestMessage(
-            to: requiredByteCount + 1)
-
-        // Create a new cache instance that uses the corrupted data persisted to disk
-        let corruptedReadingCache = try makeCache()
-
-        XCTAssertThrowsError(try corruptedReadingCache.messages()) {
-            XCTAssertEqual($0 as? CacheAdvanceError, CacheAdvanceError.fileCorrupted)
-        }
-    }
-
-    func test_messages_whenOffsetInFileAtEndOfNewestMessageIsBeyondEndOfFile_throwsFileCorrupted() throws {
-        let message: TestableMessage = "This is a test"
-        let maximumBytes = try requiredByteCount(for: [message])
-        let header = try CacheHeaderHandle(
-            forReadingFrom: testFileLocation,
-            maximumBytes: maximumBytes,
-            overwritesOldMessages: true)
-
-        func makeCache() throws -> CacheAdvance<TestableMessage> {
-            return CacheAdvance<TestableMessage>(
-                fileURL: testFileLocation,
-                writer: try FileHandle(forWritingTo: testFileLocation),
-                reader: try CacheReader(
-                    forReadingFrom: testFileLocation,
-                    maximumBytes: maximumBytes),
-                header: header,
-                decoder: JSONDecoder(),
-                encoder: JSONEncoder())
-        }
-        let writingCache = try makeCache()
-        try writingCache.append(message: message)
-
-        // Make the file corrupted by setting the offset at end of newest message to be further in the file.
-        // This could happen if a crash occurred during a write of `header.offsetInFileAtEndOfNewestMessage` on a big-endian device.
-        // Big-endian devices write the most significant digits first, meaning that if we were offsetInFileAtEndOfNewestMessage from 00001010 to 00010000, it would be possible to crash with the following bytes written to disk: 00011010.
-        // The 00011010 value is a larger value what we intended to write, which would lead to file corruption.
-        try header.updateOffsetInFileAtEndOfNewestMessage(
-            to: header.offsetInFileAtEndOfNewestMessage + 1)
-
-        // Create a new cache instance that uses the corrupted data persisted to disk
-        let corruptedReadingCache = try makeCache()
-
-        XCTAssertThrowsError(try corruptedReadingCache.messages()) {
-            XCTAssertEqual($0 as? CacheAdvanceError, CacheAdvanceError.fileCorrupted)
-        }
-    }
-
-    func test_messages_whenOffsetInFileAtEndOfNewestMessageIsBeforeEndOfNewestMessage_throwsFileCorrupted() throws {
-        let message: TestableMessage = "This is a test"
-        let maximumBytes = try requiredByteCount(for: [message])
+    func test_messages_throwsFileCorruptedWhenOffsetInFileAtEndOfNewsetMessageOutOfSync() throws {
+        let randomHighValue: UInt64 = 10_1000
```

Comment:
let's make this number less confusing (and fix an old mistake of mine) by doing either:
```suggestion
        let randomHighValue: UInt64 = 10_000
```
or:
```suggestion
        let randomHighValue: UInt64 = 101_000
```